deploy/deduplication-db-migrations/chart/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: deduplication-db-migrations
+description: |
+  A Helm chart for Kubernetes containing the dracon migrations needed for the deduplication component to run. 
+  Please check the documentation for more information
```

Comment:
can u maybe link to the relevant documentation so people know where to look?

Reply:
fixed